libs/clients/air-discount-scheme/project.json

```diff
@@ -0,0 +1,37 @@
+{
+  "root": "libs/clients/air-discount-scheme",
+  "sourceRoot": "libs/clients/air-discount-scheme/src",
+  "projectType": "library",
+  "targets": {
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "options": {
+        "lintFilePatterns": [
+          "libs/clients/air-discount-scheme/**/*.{ts,tsx,js,jsx}"
+        ]
+      }
+    },
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": ["coverage/libs/clients/air-discount-scheme"],
+      "options": {
+        "jestConfig": "libs/clients/air-discount-scheme/jest.config.js",
+        "passWithNoTests": true
```

Comment:
:sob:

Reply:
As is, the client is simply a module that exports a client with the correct middleware for a token exchange, no service.
I would like to have this test scaffold ready rather than remove it.

Is there a reason this should not be in the project file?